app/src/main/java/com/example/cristiana/workshop1/ProfileActivity.java

```diff
@@ -0,0 +1,74 @@
+package com.example.cristiana.workshop1;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.v7.app.AppCompatActivity;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.example.cristiana.workshop1.R;
+
+public class ProfileActivity extends AppCompatActivity implements View.OnClickListener {
+
+    TextView mEditText;
+    boolean mShouldReturnData;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_profile);
+
+        TextView updatedDateView = (TextView) findViewById(R.id.updatedDate);
+        SimpleDateFormat date = new SimpleDateFormat("EEE, MMM dd yyyy");
+        Date updatedAt = new Date();
+        updatedDateView.setText(date.format(updatedAt));
+
+        TextView createdDateView = (TextView) findViewById(R.id.createdDate);
+        Date createdAt = new Date("Wed, Mar 29 1995");
+        createdDateView.setText(date.format(updatedAt));
```

Comment:
Implementeaza te rog un model pentru UserData si seteaza datele din acest model in UI (vezi sample)